tests/usecase/test_emit_metrics.py

```diff
@@ -15,6 +15,22 @@ def test_read_metrics_in_logs(self):
         self.assert_string_in_recent_logs("number_of_files")
         self.assert_string_in_recent_logs("critical_logs_count")
 
+    def test_free_apps_metrics(self):
+        self.setUpCF(
+            "sample-6.2.0.mda",
+            env_vars={
+                "METRICS_INTERVAL": "10",
+                "PROFILE": "free",
+            },
+        )
+        self.startApp()
+
+        time.sleep(10)
+        self.assert_string_in_recent_logs("MENDIX-METRICS: ")
```

Comment:
This relies on the fact that metrics are emitted via stdout when `BYPASS_LOGGREGATOR` and the trends-storage-server URL environment variables are both set. 

In production, we won't be emitting metrics over stdout, so this test doesn't really accurately test the code path that we want.

(This probably isn't a big deal, but I think that this discrepancy should at least be explained in the docstring of this method)

Reply:
I left the test as it was since it was in sync with the style of the tests of this project. But since I'am refactoring more and more, it should be easier (and more desired) to introduce small scoped unit tests

Reply:
Can you please still add the above as a docstring on the `test_free_apps_metrics` method?